### PR TITLE
feat: add side nav for cloud users

### PR DESCRIPTION
**Pull request by 121watts**
### What
Adding a cloud only feature flagged "NavBox" to the side nav for the cloud specfic `/users` page. 

### Why
No one could find where it was living in the Quartz nav.

### CODE REVIEW

File: ui/src/pageLayout/containers/Nav.tsx [hunk 1]
```diff
@@ -89,6 +90,9 @@ class SideNav extends PureComponent<Props, State> {
     const feedbackLink =
       'https://docs.google.com/forms/d/e/1FAIpQLSdGJpnIZGotN1VFJPkgZEhrt4t4f6QY1lMgMSRUnMeN3FjCKA/viewform?usp=sf_link'
 
+    // Cloud
+    const cloudUsersLink = `${CLOUD_URL}/users`
```
Comment (drdelambre) [inline ui/src/pageLayout/containers/Nav.tsx]: making `/users` it's own const would make it more uniform with the convention for quartz urls like `/usage` and `/billing` and `/logout`

File: ui/src/pageLayout/containers/Nav.tsx [hunk 2]
```diff
@@ -247,6 +251,23 @@ class SideNav extends PureComponent<Props, State> {
             key="client-libraries"
           />
         </NavMenu.Item>
+        <FeatureFlag name="multiUser">
```
Comment (drdelambre) [inline ui/src/pageLayout/containers/Nav.tsx]: DAMN! look at all that conditional logic

File: ui/src/shared/constants/index.ts [hunk 3]
```diff
@@ -62,6 +62,7 @@ export const CLOUD_CHECKOUT_PATH = '/checkout'
 export const CLOUD_BILLING_PATH = '/billing'
 export const CLOUD_USAGE_PATH = '/usage'
 export const CLOUD_LOGOUT_PATH = '/logout'
+export const CLOUD_USERS_PATH = '/users'
```
Comment (drdelambre) [inline ui/src/shared/constants/index.ts]: oh.. its here, but not there

File: ui/src/shared/utils/featureFlag.ts [hunk 4]
```diff
@@ -11,6 +11,7 @@ export const OSS_FLAGS = {
 
 export const CLOUD_FLAGS = {
   deleteWithPredicate: false,
+  multiUser: false,
```
Comment (OfTheDelmer) [inline ui/src/shared/utils/featureFlag.ts]: nice

**Pull request merged**